source/extensions/filters/http/cache/cache_filter.h

```diff
@@ -88,8 +88,8 @@ class CacheFilter : public Http::PassThroughFilter,
 
   // TODO(#12901): The allowlist could be constructed only once directly from the config, instead of
```

Comment:
@jmarantz there's this TODO that reference the open issue; should I expand on this to mention the gzip filter?

Reply:
thanks; works for me.